### ISSUE TITLE
Add background images to tiles

### ISSUE THREAD

**Issue by alpo22** [seansegal/tincisnotcatan#1]
Love the game, great job. Adding background-images to the tiles would be a great enhancement.

I'd do it and make a pull request, but don't know Maven to get the app running locally to test.

**Comment by tsgoff** [comment]
You can run it locally with /build.sh and the run script or using the Docker Container: github.com/seansegal/tincisnotcatan/pull/2
Just ensure using a webserver in front of it for offloading the ssl websockets.

**Comment by seansegal** [comment]
Yes, thanks to @tsgoff you can now run the application with Docker. The pull request has been merged into master and Docker commands are available in the `README.md`.
